site/src/components/entityTypes/SchemaEditor/Inputs.tsx

```diff
@@ -27,7 +28,22 @@ export const TextInputOrDisplay: VoidFunctionComponent<{
   }, [value]);
 
   if (readonly) {
-    return <span>{value}</span>;
+    return (
+      <Box
```

Comment:
This would be better as `Typography`, which is meant to hold text (and will make it a `span` by default, as it was before)

Reply:
Have replaced `Box` with `Typography` in the latest commit: https://github.com/blockprotocol/blockprotocol/pull/167/commits/d75062cd57a5710a025383c972e66ee0f8441a74.

Now, it renders it as a `p` instead of a `span`, but I'm letting it be since the `width` property doesn't apply to a span without `display: block`.

Additionally, I've added `overflow-wrap: anywhere` to allowing breaking long unspaced words like URLs.

Reply:
Should be done in the latest commit: https://github.com/blockprotocol/blockprotocol/pull/167/commits/a7cbf6ccc8b764da321125aa70686a52895f0322